### PR TITLE
Create BLorient2083

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a record for BLorient2083, which contains a Gebra ḥemāmāt (LIT1544Gebrah), which was described by Strelcyn with great detail. 

I added some sub-IDs in the work record, that I pushed a minute ago according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2531. I identified each of the admonitions and homilies, that I newly encoded in LIT1544Gebrah in another witness (ESqmb001). However, I was not able to identify an admonition by John Chrysostomus on f. 145ra; an anonymous admonition in 147ra (both Friday morning) and an admonition by Athanasius on f. 189ra (Saturday morning) and left these without a sub ID.

Concerning a reading in f. 171va, that was entitled a 'paraphrase of Ezra' in the catalogue, I am not certain, whether I took the correct ID for the book of Ezra (and Nehemia).

Also for 183ra, I was not able to identify the correct LIT for "Song of Moses and Song of the Three Children". I took ESqmb001 as a comparison and found similar readings on folio 128vb to 129ra. However I did not find the incipits in ESqmb001 (ማኅሌት፡ ሙሴ፡)  either or it was not specific enough for an identification (ጸሎተ፡ ፫ደቅ፡).

I did not find a place or an institution with the name በበሐ፡ አቦ፡ as found in an extra (e2) on folio 48va. I would like to create a place name (LOC) and an ID for a repository (INS) as suggested in https://github.com/BetaMasaheft/Documentation/issues/2532.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 1]
```diff
@@ -2392,6 +2392,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         is named as an owner, together with her son <persName ref="PRS14420TaklaHaymanot">Takla Hāymānot</persName>.</desc> 
                                 </item>
                                 <item xml:id="e2">
+                                    <locus target="48va"/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient2083.xml:2395]: target needs `#`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 2]
```diff
@@ -0,0 +1,2470 @@
...
+                                            <locus from="116va"/>
+                                            <title ref="LIT3985Mazmura#Ps85" type="incomplete">Psalm 85:14-16</title>
+                                        </msItem>
+                                </msItem>
+                                <msItem xml:id="ms_i1.11.1.2">
+                                    <locus from="116va" to="120rb"/>
+                                    <title ref="LIT1544Gebrah#MassThur">Reading for the Mass</title>
+                                        <msItem xml:id="ms_i1.11.1.2.1">
+                                            <locus from="116va"/>
+                                            <title>Homily by Abba Sinoda</title>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient2083.xml:1301]: ʾAbbā Sinodā

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 3]
```diff
@@ -0,0 +1,2470 @@
...
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="22"/>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Neat gʷǝlḥ script, decreasing in height from 6 mm at the beginning to about 4 mm at the end of the manuscript.</desc>
+                                <date notBefore="1800" notAfter="1899" resp="PRS8999Strelcyn"/>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient2083.xml:2365]: Is the date here a typo? The origin date suggested for the manuscript is notBefore="1694" notAfter="1706", and gwelh script would usually also point to an earlier time.

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 4]
```diff
@@ -0,0 +1,2470 @@
...
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote type="bindingMaterial" xml:id="b1"><material key="wood"/></decoNote>
+                                <decoNote xml:id="b2" type="Boards">Repaired in <placeName ref="INS00001BL">British Museum</placeName>.</decoNote>
+                                <decoNote xml:id="b3" type="Cover"><material key="leather">Blind-tooled leather.</material></decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin><origDate notBefore="1694" notAfter="1706">Between 1694 and 1706</origDate></origin>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient2083.xml:2421]: Could you briefly give here the reason for the dating? I assume it refers to the mention of king Iyasu, but from how I understand the description, he is mentioned in an addition, so theoretically the manuscript could also have been produced earlier

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 5]
```diff
@@ -0,0 +1,2470 @@
...
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Neat gʷǝlḥ script, decreasing in height from 6 mm at the beginning to about 4 mm at the end of the manuscript.</desc>
+                                <date notBefore="1800" notAfter="1899" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#28va"/>
+                                    <desc type="Record">Record with the name of King <persName ref="PRS5616IyasuI">Iyāsu I</persName>.</desc>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient2083.xml:2372]: ʾIyāsu

**Review by thea-m — CHANGES_REQUESTED** [review]
Wow, so much work, congratulations!

File: LondonBritishLibrary/orient/BLorient2083.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient2083.xml:2421]: Me too, I wondered, why Strelcyn was that precise in dating the manuscript. Your idea seems to be right. I surely is the combination of ʾIyāsu I with several church dignataries in a3, that made him think, that this manuscript was written in this time frame. His judgement on the palaeography also supported this dating. However, from the catalogue, it is not clear, whether the note on 217ra, that I encoded as a3, was written down by the same or a similar contemporary hand. 

Screenshot 2024-05-07 182606

No. 1 in the following list of remarks says, that this benediction was written by a later hand. Does this mean, that the other notes mentioned in remark no. 5 are by a contemporary or by the same hand? If it were the same hand, is the encoding as an addition correct? I think, there is something, that made Strelcyn think, that these addition (a1 and a3) were made around the same time as the main text, unless he does not say explicitely. 


Screenshot 2024-05-07 182502

I think it is fair to date the manuscript 1694 to 1706 and pass the responsibility to St. Strelcyn.

Comment (thea-m) [reply LondonBritishLibrary/orient/BLorient2083.xml:2421]: Without knowing the idiosyncrasies of this catalogue, my guess would be that his note 5 does not refer to an additional note, but to supplications in the main text in which these persons are named. But this is not stated explicitly. I would however change note a3 into an e note, removing the type Record.

**Review by thea-m — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 6]
```diff
@@ -0,0 +1,2474 @@
...
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="137r" to="177r"/>
+                                    <desc type="Comment">Short notes summarizing the main events of the Passion in the upper margins of the 
+                                        readings for Good Friday.</desc>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="218ra" to="218vb"/>
+                                    <desc type="Supplication">Benediction</desc>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient2083.xml:2380]: supplications are usually scribal notes, not additiones, here we have a Prayer or a GuestText (salota burake)

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 7]
```diff
@@ -0,0 +1,2474 @@
...
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1ra"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="de">Gebra Hemamat angekauft fuer 16 Maria Theresia Thaler 
+                                        <date when="1870-08-08">d. 8 August 1870</date>. 
+                                        <persName role="owner" ref="PRS14422FMayer">F. Mayer</persName></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#28va"/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient2083.xml:2391]: From the catalogue it looks like only catalogue extras numbers 1 and 4 are additiones, the rest are colophon and supplications.  What Strelcyn has under no. 5 is a summary of information from supplications that shouls go under `origin`, not an extra

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 8]
```diff
@@ -0,0 +1,2474 @@
...
+                                    <locus target="#48va"/>
+                                    <desc type="OwnershipNote">A note in the margin indicating, that this manuscript belonged to the church of
+                                        <placeName ref="LOC7449BabahaAbbo">Babaḥa ʾAbbo</placeName>. A person with the name
+                                        <persName ref="PRS14421WaldaHanna">Walda Ḥannā</persName>, who was possibly a priest of this 
+                                        church, is also mentioned.</desc> 
+                                    <q xml:lang="gez">ዝመጽሐፍ፡ ዘደብረ፡ በበሐ፡ አቦ፡</q>
+                                </item>
+                                <item xml:id="e4">
+                                    <locus target="#122vb"/>
+                                    <desc type="OwnershipNote"><persName role="owner" ref="PRS14419Maryamawit">Māryāmāwit</persName>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient2083.xml:2404]: I assume they are named in a supplication; the information can be provided under `provenance`

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 9]
```diff
@@ -0,0 +1,2474 @@
...
+                                        is named as an owner, together with her son <persName ref="PRS14420TaklaHaymanot">Takla Hāymānot</persName>.</desc> 
+                                </item>
+                                <item xml:id="e5">
+                                    <locus target="#217ra"/>
+                                    <desc>Supplication with the names of Patriarch
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient2083.xml:2409]: supplications are usually by the scribe and not additions, this is relevant for dating/origin

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient2083.xml (hunk 7 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient2083.xml:2391]: Shall I write into `<origin></origin>` or into `<origDate notBefore="1694" notAfter="1706"></origDate>`?

Comment (eu-genia) [reply LondonBritishLibrary/orient/BLorient2083.xml]: you can put it into `origDate`

Comment (eu-genia) [reply LondonBritishLibrary/orient/BLorient2083.xml]: supplications and colophons should be in most cases (unless these are clear additions) encoded with the relevant `msItem`s https://github.com/BetaMasaheft/Documentation/issues/1003
https://github.com/BetaMasaheft/Documentation/issues/895

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 10]
```diff
@@ -2421,7 +2406,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                     </physDesc>
                     <history>
-                        <origin><origDate notBefore="1694" notAfter="1706">Between 1694 and 1706</origDate></origin>
+                        <origin><origDate notBefore="1694" notAfter="1706">Between 1694 and 1706</origDate>.
+                            King <persName ref="PRS5616IyasuI">ʾIyāsu I</persName> is mentioned on <locus target="#28va"/>, 
+                            while Patriarch <persName ref="PRS5711JohnXVI"><roleName type="title">ʾAbbā</roleName> Yoḥannǝs</persName> 
+                            and Metropolitan <persName ref="PRS6777Marqos"><roleName type="title">ʾAbbā</roleName> Mārqos</persName>    
+                            are mentioned on <locus target="#217ra"/>.</origin>
+                        <provenance><persName role="owner" ref="PRS14419Maryamawit">Māryāmāwit</persName> is named as an owner, 
+                            along with her son <persName ref="PRS14420TaklaHaymanot">Takla Hāymānot</persName>. </provenance>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient2083.xml:2415]: `on <locus target="#122vb"/>`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient2083.xml [hunk 11]
```diff
@@ -0,0 +1,2466 @@
...
+                                                <msItem xml:id="ms_i1.8.1.1">
+                                                    <locus from="70ra" to="70vb"/>
+                                                    <title ref="LIT1544Gebrah#WedNightFirst">Readings for the first night hour</title>
+                                                    <note>Indicated by <foreign xml:lang="gez">በቀዳሚት፡ ሰዓተ፡ ሌሊት፡ ዘረቡዕ፡</foreign>.</note>
+                                                        <msItem xml:id="ms_i1.8.1.1.1">
+                                                            <locus from="70ra"/>
+                                                            <title ref="LIT1373Bookof" type="incomplete">Ezekiel 22:17-32</title>
+                                                        </msItem>
+                                                        <msItem xml:id="ms_i1.8.1.1.2">
+                                                            <locus target="70rb"/>
```
Comment (eu-genia) [inline LondonBritishLibrary/orient/BLorient2083.xml:750]: target needs #

**Review by eu-genia — APPROVED** [review]

**Pull request merged**